app.js

```diff
@@ -0,0 +1,9 @@
+const express = require('express')
+const {getApiDocu} = require('./db/Controllers/api.controller')
+
+const app = express()
+
+app.get('/api', getApiDocu)
```

Comment:
I'd suggest getApi is a good enough name for this function - not sure everyone would understand what docu refers to and we aren't actually getting the document - just the data!